src/crazyflie/lib/src/bbzcrazyflie.c

```diff
@@ -85,6 +85,20 @@ void bbz_func_call(uint16_t strid) {
     }
 }
 
+void handleOutgoingRadioMessage() {
+    Message *msg = message_tx();
+    while (msg!=NULL) {
+        msg = message_tx();
+        P2PPacket pk;
+        pk.port = BROADCAST_PORT;
+        // Max size possible for a broadcast packet
+        pk.size = PAYLOAD_MAX_SIZE - 2;
```

Comment:
Can you say why -2?

Reply:
It's actually -1 because the SyslinkPacket packet length has to be <=SYSLINK_MTU. I will change the PAYLOAD_MAX_SIZE value.